### ISSUE TITLE
ConnectionClosedException

### ISSUE THREAD

**Issue by pete911** [RallyTools/RallyRestToolkitForJava#1]
Hi, 
we get "org.apache.http.ConnectionClosedException: Premature end of Content-Length delimited message body" exception when calling create method on RallyRestApi

I've already fixed it by subclassing RallyRestApi, but would be nice if you could fix it directly in that class. 

this is the file - https://github.com/RallyTools/RallyRestToolkitForJava/blob/master/src/com/rallydev/rest/RallyRestApi.java

and you need to replace line 256 with "return new Scanner(entity.getContent()).useDelimiter("\A").next();"

Basically EntityUtils throws that exception, but calling getContent directly on entity is fine.

thanks.
